lib/linter/linter.js

```diff
@@ -444,7 +444,7 @@ function normalizeEcmaVersion(ecmaVersion) {
     return ecmaVersion >= 2015 ? ecmaVersion - 2009 : ecmaVersion;
 }
 
-const eslintEnvPattern = /\/\*\s*eslint-env\s(.+?)\*\//gu;
+const eslintEnvPattern = /\/\*\s*eslint-env\s([\s\S]+?)\*\//gu;
```

Comment:
```suggestion
const eslintEnvPattern = /\/\*\s*eslint-env\s(.+?)\*\//gsu;
```

It might look nicer with the new syntax (the `s` flag)?